repository/src/main/java/org/alfresco/service/cmr/tagging/TaggingService.java

```diff
@@ -362,6 +374,13 @@ default List<Pair<String, NodeRef>> createTags(StoreRef storeRef, List<String> t
     {
         return Collections.emptyList();
     }
+
+    /**
+     *
+     * @param storeRef
+     * @return a map with each tag name and its count

```

Comment:
Minor - can come in another iteration.
```suggestion
     * @return a map with each tag name and its usage count

```